tests/test_ccc.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from tests.utils import file_response
+from documenters_aggregator.spiders.ccc import CccSpider
+
+def test_tests():
+    print('Please write some tests for this spider or at least disable this one.')
+    assert False
+
+
+"""
+Uncomment below
```

Comment:
You'll need to merge latest master which the blockers from yesterday and then write a few tests for this.